### ISSUE TITLE
manual/javaGuide/_Sidebar.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#934]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/_Sidebar.md
- diffs between [2.2.0/manual/javaGuide/_Sidebar.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/_Sidebar.md) and [2.3.5/manual/javaGuide/_Sidebar.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/_Sidebar.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/_Sidebar.md   2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/_Sidebar.md  2014-07-28 12:26:38.000000000 +0900
@@ -1,10 +1,11 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 ### Main concepts

 - [[HTTP programming | JavaActions]]
 - [[Asynchronous HTTP programming | JavaAsync]]
 - [[The template engine | JavaTemplates]]
 - [[HTTP form submission and validation | JavaForms]]
-- [[Working with JSON | JavaJsonRequests]]
+- [[Working with JSON | JavaJsonActions]]
 - [[Working with XML | JavaXmlRequests]]
 - [[Handling file upload | JavaFileUpload]]
 - [[Accessing an SQL database | JavaDatabase]]
@@ -14,12 +15,8 @@
 - [[Internationalization | JavaI18N]]
 - [[The application Global object | JavaGlobal]]
 - [[Testing your application | JavaTest]]
+- [[Logging|JavaLogging]]

 ### Advanced Topics

 - [[Extending Play|JavaPlugins]]
-
-### Tutorials
-
-- [[Your first application | JavaTodoList]]
-- [[Zentask | JavaGuide1]]

```

**Comment by garbagetown** [comment]
やります :hand:

**Comment by garbagetown** [comment]
merged at #1054
